src/HypothesisClient/Credentials/Credentials.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace eLife\HypothesisClient\Credentials;
+
+use Serializable;
+
+class Credentials implements CredentialsInterface, Serializable
```

Comment:
Looks like a value object so no need for an interface (the naming here is a giveaway; `Interface` as a suffix is a bad idea).